modules/nf-core/sourmash/compare/main.nf

```diff
@@ -14,21 +14,22 @@ process SOURMASH_COMPARE {
     val save_csv
 
     output:
-    tuple val(meta), path("*comp.npy.labels.txt"), path("*comp.npy"), optional:true, emit: matrix
-    path "*comp.csv"       , optional:true, emit: csv
-    path "versions.yml"    , emit: versions
+    tuple val(meta), path("*comp.npy")           , optional:true, emit: matrix
+    tuple val(meta), path("*comp.npy.labels.txt"), optional:true, emit: labels
+    path "*comp.csv"                             , optional:true, emit: csv
+    path "versions.yml"                                         , emit: versions
```

Comment:
```suggestion
    path "*comp.csv"                             , optional:true, emit: csv
    path "versions.yml"                          , emit: versions
```

Reply:
I've moved the emits first, optional later to make it look nicer in my eyes